src/chainparams.cpp

```diff
@@ -174,6 +174,13 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nWindowSize = 4032;
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nThreshold = 3226; // 80% of 4032
 
+        // Deployment of InstantSend autolocks
+        consensus.vDeployments[Consensus::DEPLOYMENT_ISAUTOLOCKS].bit = 3;
```

Comment:
Bump this to bit 4 pls, bit 3 is already used for DIP0003 on devnet/regtest.

Btw, @codablock any reason why we don't have DIP0003 bit settings for mainnet/testnet?